### DISCUSSION TITLE
update command with exclude option

### DISCUSSION THREAD

**Discussion by bymayo** [craftcms/cms#15795] [Ideas] [answered]
It would be good if when we run `craft update craft blitz sprig` etc, we had the option to exclude a plugin also by doing something like

`craft update all --exclude=sprig,blitz`

The main reason for this is sometimes we have plugins that we don't want to update, if for example it's running a new major version we haven't got time to update to just yet that we know will have breaking changes, but still want all other craft/plugin updates.

It can sometimes be very time consuming running `craft update` if the list of plugins we do want to update is lengthy!

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Just added an `--except` option for Craft 4.13 and 5.5, as well as `--minor-only` and `--patch-only` options. (#15829)

**Reply by brandonkelly** [reply]
Craft 4.13.0 and 5.5.0 are out with those new `update` options 🎉
